os/board/rtl8721csm/src/rtl8721csm_boot.c

```diff
@@ -175,9 +178,14 @@ void amebad_mount_partions(void)
 {
 #if defined(CONFIG_FLASH_PARTITION) || defined(CONFIG_FS_ROMFS)
 	int ret;
+	struct mtd_dev_s *mtd;
```

Comment:
You can declare this into the line 185. Sometimes(not enabled CONFIG_FLASH_PARTITION, but enabled CONFIG_FS_ROMFS), it can be build error.

Reply:
Modified